pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -1083,32 +1083,44 @@ func TestDeploymentTimedOut(t *testing.T) {
 		{
 			name: "no progressDeadlineSeconds specified - no timeout",
 
-			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, null, timeFn(1, 9)),
-			nowFn:    func() time.Time { return timeFn(1, 20) },
+			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, null, timeFn(1, 9, 0)),
+			nowFn:    func() time.Time { return timeFn(1, 20, 0) },
 			expected: false,
 		},
 		{
 			name: "progressDeadlineSeconds: 10s, now - started => 00:01:20 - 00:01:09 => 11s",
 
-			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, &ten, timeFn(1, 9)),
-			nowFn:    func() time.Time { return timeFn(1, 20) },
+			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, &ten, timeFn(1, 9, 0)),
+			nowFn:    func() time.Time { return timeFn(1, 20, 0) },
 			expected: true,
 		},
 		{
 			name: "progressDeadlineSeconds: 10s, now - started => 00:01:20 - 00:01:11 => 9s",
 
-			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, &ten, timeFn(1, 11)),
-			nowFn:    func() time.Time { return timeFn(1, 20) },
+			d:        deployment(extensions.DeploymentProgressing, v1.ConditionTrue, &ten, timeFn(1, 11, 0)),
+			nowFn:    func() time.Time { return timeFn(1, 20, 0) },
 			expected: false,
 		},
+		{
+			name: "handle nanoseconds properly #1",
```

Comment:
Would you add things like "progressDeadlineSeconds: 10s, now - started => 00:01:20 - 00:01:11 => 9s" in the name?